virttest/virt_vm.py

```diff
@@ -677,9 +677,12 @@ def get_address(self, index=0):
             # get valid IPADDR
             if arp_ip:
                 if not utils_net.verify_ip_address_ownership(arp_ip, macs, devs=devs):
-                    self.address_cache[nic.mac.lower()] = None
-                    arp_ip = None
-
+                    if self.address_cache.get(nic.mac.lower()) == arp_ip:
```

Comment:
Instead of calling `nic.mac.lower()` each every time can this be put in to a variable and then use?
`mac_lower = nic.mac.lower()`